test/Tool/signbits1-invalid.opt

```diff
@@ -0,0 +1,11 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: Invalid
+%0:i32 = var
+%1:i32 = var
```

Comment:
I have updated the parser to accept s=1. Earlier I didn't allow it to accept s=0, 1.